src/common/options.cc

```diff
@@ -3655,7 +3655,7 @@ std::vector<Option> get_global_options() {
     .set_description("Key value database to use for bluestore"),
 
     Option("bluestore_allocator", Option::TYPE_STR, Option::LEVEL_DEV)
-    .set_default("stupid")
+    .set_default("bitmap")
```

Comment:
Perhaps for the intiial backport we leave teh default as stupid and let some users opt in?

I'm a bit worried there will be some weird pathological allocation behavior that we don't see in qa but happens with production workloads.  The new allocator is very stable in qa but it hasn't actually ever been deployed in a real cluster yet.

Reply:
yeah, makes sense. Reverting...